src/openloco/town.h

```diff
@@ -35,12 +35,20 @@ namespace openloco
         uint8_t pad_02[0x06 - 0x02];
         uint16_t flags;
         uint8_t pad_08[0x30 - 0x08];
-        uint32_t population; // 0x30
+        int32_t population; // 0x30
```

Comment:
Can the population be negative?

Reply:
This change was done due to this line, I think:
```
int32_t newPop = std::max(currTown.population - currTown.unk_14C, 0);
```
From this line, we can also infer it shouldn't be negative, though. Just take care not to underflow.